indico/modules/events/editing/service_test.py

```diff
@@ -40,6 +41,26 @@ def _assert_yaml_snapshot(snapshot, data, name):
     assert_yaml_snapshot(snapshot, data, name, strip_dynamic_data=True)
 
 
+def test_service_handle_enabled(dummy_event, mocked_responses, snapshot):
+    from indico.modules.events.editing.service import service_handle_enabled
+    resp = mocked_responses.put(f'{MOCK_SVC}/event/dummy')
+    service_handle_enabled(dummy_event)
+    req_payload = json.loads(resp.calls[0].request.body)
+    _assert_yaml_snapshot(snapshot, req_payload, 'service_enabled.yml')
+
+
+def test_service_handle_disconnected(dummy_event, mocked_responses):
+    from indico.modules.events.editing.service import service_handle_disconnected
+    mocked_responses.delete(f'{MOCK_SVC}/event/dummy')
+    service_handle_disconnected(dummy_event)
+
+
+def test_service_get_status(dummy_event, mocked_responses):
+    from indico.modules.events.editing.service import service_get_status
+    mocked_responses.get(f'{MOCK_SVC}/event/dummy', json={})
+    service_get_status(dummy_event)
```

Comment:
Do we also have some response data here to compare against a snapshot?

Reply:
`service_get_status` doesn't do anything with the response, so it wouldn't be particularly useful. Sure, we could test that it returns it in 'status' but it feels a bit pointless.

(the main goal here is to test that we don't suddenly send different data to the service)

Reply:
Ok! It was just a bit a weird to me to see a test without an assert

Reply:
`responses` automatically fails tests if any mocked url has not been called ;)